core/node/libp2p/rcmgr_defaults.go

```diff
@@ -72,33 +72,28 @@ Run 'ipfs swarm limit all' to see the resulting limits.
 			Memory: int64(maxMemory),
 			FD:     int(numFD),
 
-			// By default, we just limit connections on the inbound side.
+			// At least as of 2023-01-25, it's possible to open a connection that 
+			// doesn't for any memory usage with the libp2p Resource Manager/Accountant
+			// (see https://github.com/libp2p/go-libp2p/issues/2010#issuecomment-1404280736).
+			// As a result, we can't curretly rely on Memory limits to full protect us.
+			// Until https://github.com/libp2p/go-libp2p/issues/2010 is addressed, 
+			// we take a proxy now of restricting to 1 inbound connection per MB.
+			// Note: this is more generous than go-libp2p's default autoscaled limits which do
+			// 64 connections per 1GB
+			// (see https://github.com/libp2p/go-libp2p/blob/master/p2p/host/resource-manager/limit_defaults.go#L357 ).
+			ConnsInbound:  maxMemory >> 20, // Steve: I assume this isn't valid Go code.
```

Comment:
```suggestion
			ConnsInbound:  int(maxMemory >> 20), // Steve: I assume this isn't valid Go code.
```
or maybe a tiny bit easier to read:
```suggestion
			ConnsInbound:  int(maxMemory /1024), // Steve: I assume this isn't valid Go code.
```

Reply:
Sure, good idea.  Although I assume I need to divide by (1024 * 1024) to go from bytes to MB.

I updated.